docs/how_tos/i18n.rst

```diff
@@ -74,6 +79,16 @@ Set up Transifex
 
 #. Log in as a user that can create new resources under the `edx-platform project <https://www.transifex.com/open-edx/edx-platform/>`_. Create a new resource with the same name as your repo. Languages are determined at the project level, so you will automatically get all 70+ languages even though you will probably only be actually using a few of them.
 
+   To create the resource, you may need to drag and drop your generated ``transifex_input.json`` file (see `Set up the pipeline tools`_) into the edx-platform project in Transifex.
```

Comment:
This references the 3rd step in the 2nd step. Should we rearrange the order?

Reply:
I'll leave this to @albemarle to enhance if and when she wants to.  It's complex when you get there.